delivery_recipe/daily_standup.md

```diff
@@ -0,0 +1,21 @@
+# Daily Standup
+
+The daily standup is an important meeting that we hold every day during each sprint.
+
+## Who should attend?
+
+The meeting must be attended by the Delivery Team, the Service Delivery Manager and the Product Owner. Other people can join but only as observers.
+
+## The purpose
+
+The reason we have a daily standup is due to the fast paced, ever changing nature of the work we deliver. The daily standup is the opportunity for the each member of the team to share the details of what they're working on, what they have planned for that day and to raise any problems they're encountering. The meeting also serves to communicate the progress with the Product Owner and to seek help if any members of the delivery team have encountered problems that need to be resolved.
+
+## The process
+
+Each member of the delivery team takes turns to discuss the stories they worked on the previous day, what they plan to work on today and to raise any issues that they’ve encountered.
+
+This is a quick meeting designed as a progress checkpoint, to request any support that’s needed and to keep an eye on the overall picture of the sprint.
+
+If there are any issues raised that require more discussion or support then we agree who’s to be involved and those discussions take place in a separate meeting.
+
+The meeting is to take no more than 15 minutes. We don’t involved people in further discussions unless we need to.
```

Comment:
I think it's worth expanding on the 'culture of the meeting' a bit...
- always in the same place at the same time
- speak briefly and focus on important stuff, not just busywork
- this is not the team reporting to a manager, it is them synchronising with each other, others may observe but not contribute
- any discussions that develop should be parked and a time agreed to address them instead
- the most important part of the meeting is the blockers
- scrummaster or someone deputising for them facilitates the meeting to keep things moving, but remember the team is not reporting to the scrummaster